assets/default_bundle/drp_munich_village.csv

```diff
@@ -1,1629 +1,1225 @@
 # =======================================
-#           drp_munich_autobahn                 
+#           drp_munich_autobahn
 # =======================================
 # iperf3 DataRatePattern (UDP)
 # This DRP was recorded on a city/village street
 # in the wieder area of munich.
-# 
```

Comment:
Please add a line describing that it was modified from a 'real' pattern

Reply:
done

Reply:
Some more typos in the comments have been corrected.